src/osd/OSDMap.cc

```diff
@@ -5987,7 +5987,12 @@ void OSDMap::check_health(CephContext *cct,
   }
 
   // OSD_UPGRADE_FINISHED
-  // none of these (yet) since we don't run until luminous upgrade is done.
+  if (auto require_release = pending_require_osd_release(); require_release) {
+    ostringstream ss;
+    ss << "all OSDs are running " << require_release << " or later but"
```

Comment:
that should be *require_release, I think

Reply:
And probably the following is sufficient:

```
if (auto require_release = pending_require_osd_release()) {
  ...
}
```